corehq/apps/case_search/utils.py

```diff
@@ -65,13 +67,22 @@ def _assemble_optional_search_params(self):
         self._add_daterange_queries()
         self._add_case_property_queries()
 
+    def _validate_param_value(self, key, value):
+        if isinstance(value, list):
+            raise CaseFilterError(
+                _("Multiple xpath queries for a query param is not supported"),
```

Comment:
this message needs to be more generic now:
```suggestion
                _("Multiple values not supported"),
```

Reply:
Fixed it, thanks!